homeworks/lesson2/app/src/components/login/login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {Button, FormGroup, FormControl, ControlLabel} from 'react-bootstrap';
+import './login.css';
+
+class LoginForm extends Component {
```

Comment:
Why in another code files you are using React.Component and in this case you are used Component?
Please be more consistent in your code.